apps/src/code-studio/pd/workshop_dashboard/components/facilitator_list_form_part.jsx

```diff
@@ -105,15 +110,19 @@ export default class FacilitatorListFormPart extends React.Component {
     }
 
     let addButton = null;
+    // Only show add button on the last row, when we're below the max,
+    // when it's not a placeholder (id -1), and we have at least one available to add
     if (i === facilitators.length - 1 && facilitators.length < MAX_FACILITATORS &&
-      this.props.facilitators[i].id > 0 && filteredAvailableFacilitators.length > 1 ) {
+      this.props.facilitators[i].id > 0 && filteredAvailableFacilitators.length >= 1 ) {
```

Comment:
There might be another bug here - if the placeholder id is -1, shouldn't the third condition here be `this.props.facilitators[i] >= 0` not `> 0`?

Reply:
Rails ids start at 1, so in this case either way will work. I'd be happy to change it to >= 0 if you think that's more readable.

Reply:
TIL. LGTM.

Reply:
Or maybe it's due to the underlying [MySql auto increment](https://dev.mysql.com/doc/refman/5.7/en/example-auto-increment.html), but anyway this is the behavior - our ids always start with 1.